### ISSUE TITLE
system reboot mining cn-heavy - does watchdog reboot computer?

### ISSUE THREAD

**Issue by newmzy** [technobyl/CryptoDredge#27]
I have contacted your email and posted on bitcointalk.org but I may as well post here too....

I am running windows 8.1 x64 cryptodredge 0.9.3

1 x 1080 3 x 1070

trying to mine loki with cryptonight-heavy

the hash-rates are great compared to xmr-stak (which runs stable for days) but if I use exactly the same setting, overclocks etc my computer does a hard reboot after a random period from 5 - 30 minutes.

I have tried reverting to P-state 2, lowering overclocks and updating to a newer (but not latest) driver.

Does your watchdog reboot the machine or just restart (or stop) the miner?

Has anyone else reported full system reboots?

**Comment by platinum4** [comment]
No, and current version is v0.23.0 now, please close this old issue @newmzy
